ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.scss

```diff
@@ -173,19 +173,12 @@
 .feeds {
     display: grid;
     grid-template-columns: 1fr 1fr;
-    grid-gap: 1rem;
```

Comment:
@fat23cat did I miss any changes in this file?

Reply:
No, you didn't. Looks good!